src/models/volunteers.js

```diff
@@ -0,0 +1,60 @@
+const { sequelizeConnection, Sequelize, DataTypes } = require('./index.js');
+const survey = require('./survey');
+const dailySurveys = require('./dailySurveys');
+
+const newVolunteers = sequelizeConnection.define('newVolunteers', {
```

Comment:
why is this called 'new'? just volunteers?

Reply:
We're doing that just because we imported the model as volunteers already, but that naming convention can be changed if it's your preference!